File: TrabajoFinGrado/Backend/Backend-Automoviles/src/main/kotlin/com/example/backendautomoviles/filters/ReservaFilter.kt
package com.example.backendautomoviles.filters


class ReservaFilter(
    val id : String?,
    val clienteId : String?,
    val automovilId : String?,
    val fechaInicio : String?,
    val fechaFinal : String?,
    val costo : Double?,
    val recogidoPorCliente : Boolean?,
) {
}

File: TrabajoFinGrado/Frontend/Frontend-Web/src/app/models/reserva/reserva-filter.ts
export class ReservaFilter {
    public id: string | null;
    public automovilId: string | null;
    public clienteId: string | null;
    public fechaInicio: string| null;
    public fechaFin : string | null;
    public costo : string | null;
    public recogidoPorCliente : boolean | null;
  
    constructor() {
      this.id = '';
      this.automovilId = '';
      this.clienteId = '';
      this.fechaInicio = '';
      this.fechaFin = '';
      this.costo = '';
      this.recogidoPorCliente = false;
    }
  }
  